dragonfly/implementations/roach_daq_run_interface.py

```diff
@@ -210,49 +212,43 @@ def _stop_data_taking(self):
                 logger.info('Psyllid still running. Telling it to stop the run')
                 self.provider.cmd(self.psyllid_interface, 'stop_run', payload = self.payload_channel)
         except core.exceptions.DriplineError:
-            logger.error('Getting Psyllid status at end of run failed')
+            logger.critical('Getting Psyllid status at end of run failed')
             logger.info('unblock channel')
             payload = {'channel': self.channel_id}
             try:
-                self.provider.cmd(self.daq_target, 'unblock_channel', payload=payload)
+                self.provider.cmd(self.daq_target, 'unblock_channel', payload = payload)
             finally:
                 raise core.exceptions.DriplineGenericDAQError('Error at end of run')
         except Exception as e:
             logger.error('Local error at end of run')
             logger.info('unblock channel')
             payload = {'channel': self.channel_id}
             try:
-                self.provider.cmd(self.daq_target, 'unblock_channel', payload=payload)
+                self.provider.cmd(self.daq_target, 'unblock_channel', payload = payload)
             finally:
                 raise e
         else:
             payload = {'channel': self.channel_id}
-            self.provider.cmd(self.daq_target, 'unblock_channel', payload=payload)
+            self.provider.cmd(self.daq_target, 'unblock_channel', payload = payload)
             if self.status_value==None:
                 raise core.exceptions.DriplineGenericDAQError('Psyllid must have crashed during run')
 
 
-    def adopt_new_psyllid_instance(self):
-        self.provider.cmd(self.psyllid_interface, 'get_acquisition_mode', payload = self.payload_channel)
-        self._finish_configure()
-
-
     def stop_psyllid(self):
```

Comment:
return something like 'psyllid stopped'

Reply:
ok

Reply:
as above, don't return anything; our convention is to use exceptions and reply return codes to indicate success failure.